samples/expert-finder-js/README.md

```diff
@@ -0,0 +1,92 @@
+---
+page_type: sample
+description: This sample demonstrates how to use a copilot plugin that can be used to search candidates for job. This sample can also be used as a search based messaging extension.
+products:
+- office-teams
+- copilot-m365
+languages:
+- javascript
+---
+
+# Expert Finder Copilot Plugin
+
+## Interaction with app
+
+ ![expert-finder-gif](images/expert-finder.gif)
+
+## Try it yourself - experience the App in your Microsoft Teams client
```

Comment:
Will it work directly without creating table storage?

Reply:
added steps for creating storage account